Branch-SDK/Branch-SDK/BranchCreditHistoryRequest.m

```diff
@@ -63,11 +64,11 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     }
     
     for (NSMutableDictionary *transaction in response.data) {
-        if (transaction[@"referrer"] == [NSNull null]) {
-            [transaction removeObjectForKey:@"referrer"];
+        if ([transaction[BRANCH_RESPONSE_KEY_REFERRER] isEqual:[NSNull null]]) {
```

Comment:
Agreed. isEqual is safer than ==, regardless the fact that [NSNull null] is singleton.